tests/functional/pod_and_daemons/test_csiaddon_daemons_pods.py

```diff
@@ -214,3 +214,49 @@ def test_csi_addon_pod_restart(self):
         assert wait_for_pods_to_be_running(
             namespace=namespace, pod_names=csi_addon_pod_names_list
         ), "CSI-addons pod didn't came up is running sattus "
+
+    def test_csi_addons_pod_crash_recovery(self):
```

Comment:
Do we need to skip it for external mode? And can we run the verification job on all supported UPI/IPI platforms?

Reply:
[Job PASSED](https://url.corp.redhat.com/3b91b6a). Test Passed on external mode cluster too. Marking this comment as resolved.